7thHeaven.Code/FFNxConfigManager.cs

```diff
@@ -235,6 +235,9 @@ public void OverrideInternalKeys()
             _toml["ffmpeg_video_ext"] = "avi";
             _toml["mod_path"] = "mods/Textures";
             _toml["direct_mode_path"] = "direct";
+
+            if (Directory.EnumerateFiles(Path.Combine(Sys.InstallPath, "music/vgmstream"), "*.ogg").Any())

```

Comment:
It will be automatically set to false as this sub is called when FFNx is upgraded or when you hit Reset Defaults ( which all it does is copying the original FFNx.toml and it applies these overrides on top ).

Reply:
Oh sorry, I thought this was the code section which is executed before running the game

Reply:
No worries, thanks though for raising a good point nevertheless :)